tiled/server/router.py

```diff
@@ -617,3 +616,21 @@ async def put_dataframe_full(
             status_code=405, detail="This path cannot accept dataframe data."
         )
     return json_or_msgpack(request, None)
+
+
+@router.put("/node/metadata/{path:path}")
+async def put_metadata(
+    request: Request,
+    body: schemas.PutMetadataRequest,
+    entry=Security(entry, scopes=["write:data", "write:metadata"]),
```

Comment:
The scopes listed on some of the writing routes were too broad, which I just fixed in existing routes in 5c79c37. This route should only require `write:metadata`.

```suggestion
    entry=Security(entry, scopes=["write:data", "write:metadata"]),
```

Reply:
Fixed in new commit